pkg/controller/atlasdatafederation/private_endpoint.go

```diff
@@ -1,64 +1,103 @@
 package atlasdatafederation
 
 import (
-	"context"
+	"fmt"
 
 	"github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/translation/datafederation"
-
-	"github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/set"
 	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api"
 	akov2 "github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1"
 	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/controller/workflow"
 )
 
-func (r *AtlasDataFederationReconciler) ensurePrivateEndpoints(ctx *workflow.Context, project *akov2.AtlasProject, dataFederation *akov2.AtlasDataFederation, service datafederation.DatafederationPrivateEndpointService) workflow.Result {
+func (r *AtlasDataFederationReconciler) ensurePrivateEndpoints(ctx *workflow.Context, service datafederation.DatafederationPrivateEndpointService, project *akov2.AtlasProject, dataFederation *akov2.AtlasDataFederation) workflow.Result {
```

Comment:
I would not change order of params, it is an internal API and does not matter much, but still there is no reason for the change AFAIK

Reply:
see https://github.com/mongodb/mongodb-atlas-kubernetes/pull/1929#discussion_r1846084044

Reply:
I actually suggest to change the variable ordering, it makes the signature clearer.